### DISCUSSION TITLE
TTN Recieving Accept Join-Request but no data Payloads

### DISCUSSION THREAD

**Discussion by grahamrobertslx** [cyberman54/ESP32-Paxcounter#821] [Q&A]
HI There,

I have got my Paxcounter talking with TTNv3 but im not recieving any data back. I get Accept join Request but no Payload Data.

Where might i be going wrong?

G

**Comment by cyberman54** [comment]
Please consider these general single points of failure:
- Your radio coverage is too weak -> make sure you are close to a TTN v3 gateway
- You are trying to join via a TTN v2 gateway
- You are using a single or dual channel gateway (like Dragino etc.) -> this won't work
- You are using wrong DIO pin mappings for your board
- You are using some kind of payload converter/decoder
- ...
